### ISSUE TITLE
aFem Manual Processing, Part 1 (GRA, CCS, PWG)

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/CORRECTIONS#160]
Have started to browse https://github.com/sanskrit-lexicon/CORRECTIONS/blob/644736676a80f7d961d6ef6432a4bfeaaef7efd3/afem/readme.md 

gra_afem
ásita, fem. ásiknī - references another, correct form; no change

ccs_afem
atisṛṣiṭa -> atisṛṣṭī (OCR error)
![atisfziwa](https://cloud.githubusercontent.com/assets/80761/11253760/7b294d9a-8e4e-11e5-99c2-1f94fa3d9ba6.PNG)

ariṣṭa -> ariṣṭatātī¦ f. Unversehrtheit.  (OCR error)
![arizwa](https://cloud.githubusercontent.com/assets/80761/11254191/6f0df8c4-8e50-11e5-926c-ce0505aafcd4.PNG)

ūrṇa -> ūrṇā  (OCR error)
![urra](https://cloud.githubusercontent.com/assets/80761/11254249/afeb78da-8e50-11e5-9369-14bb39256e0c.PNG)

kulapratiṣṭha -> kulapratiṣṭhā  (OCR error)
![kulapratizwa](https://cloud.githubusercontent.com/assets/80761/11254297/ebe6ca7e-8e50-11e5-882e-56d658995512.PNG)

kṛṣṭa [L=5343] -> kṛṣṭī (OCR error)
![kfzwa](https://cloud.githubusercontent.com/assets/80761/11254341/25dc1f72-8e51-11e5-8e06-7acef09c359d.PNG)

cikyāna¦ f. 2. ci -> cikyāna¦ s. 2. ci
![cikyana](https://cloud.githubusercontent.com/assets/80761/11254387/705a5ffa-8e51-11e5-8678-ccdef11ee0d7.PNG)

jijīviṣa -> jijīviṣā (OCR error)
![jijiviza](https://cloud.githubusercontent.com/assets/80761/11255020/0816cf10-8e55-11e5-9806-0a6640ed0632.PNG)

**Comment by drdhaval2785** [comment]
@gasyoun

Try afem.html

Linked to the webpage and pdf

**Comment by gasyoun** [comment]
@drdhaval2785 it was there before, can't see it anymore. Can you?

**Comment by drdhaval2785** [comment]
Maybe I missed to commit properly.
Here it is - https://github.com/sanskrit-lexicon/CORRECTIONS/blob/master/afem/afem.html
Make a local copy and open in local browser.
Not putting up on github.io.
Already too cluttered.

**Comment by drdhaval2785** [comment]
@gasyoun Good to know that most of the entries are errors.
Hope these 251 word list give us at least 100 errors.

**Comment by gasyoun** [comment]
@drdhaval2785 depends on source. `CCS` are for sure.

tvaca [L=9186] -> tvac  (OCR error)
![tvac](https://cloud.githubusercontent.com/assets/80761/11287414/5738df48-8f2c-11e5-9e39-b58714aca75c.PNG)

**Comment by gasyoun** [comment]
durvārtta -> durvārttā  (OCR error)
![durvartta](https://cloud.githubusercontent.com/assets/80761/11287457/b7ed922a-8f2c-11e5-875f-e3ec545d8f92.PNG)

**Comment by gasyoun** [comment]
devayoṣa -> devayoṣā (OCR error)
![devayoza](https://cloud.githubusercontent.com/assets/80761/11287471/eb909c6c-8f2c-11e5-9463-a0d285db1e18.PNG)

**Comment by gasyoun** [comment]
>devāgāra [L=10379] [p= 183-1] 	devāgāra¦ f. Göttergeschoss. - no such entry.
If I look above 
>devaheti [L=10378] [p= 183-1] 	devaheti¦ f. u. devahūya n. Götteranrufung.
is wrong as well. If has the meaning which the word above should have.
It should have been
>devāgāra [L=10379] [p= 183-1] 	devāgāra¦ m. n. Götterhaus, Tempel.
So the meanings are wrong in several entries, "belonging" to the entry above.
> devasad [L=10368] [p= 183-1] 	devasad¦ unter den Göttern gehört o. erhört.
might be the first one wrong. So half the column has meanings it should not have in OCR.
So
devāgāra¦ f. Göttergeschoss. -> devāgāra¦ m. n. Götterhaus, Tempel.

![deva](https://cloud.githubusercontent.com/assets/80761/11287614/1c5e2f70-8f2e-11e5-986c-d8271a7947b8.PNG)

**Comment by gasyoun** [comment]
nīṣpada ->  nīṣpad (OCR error)
![nizpada](https://cloud.githubusercontent.com/assets/80761/11287654/6a57a7ba-8f2e-11e5-8c85-c1a7cf8fd537.PNG)

**Comment by gasyoun** [comment]
paṭahaghoṣaṇa -> paṭahaghoṣaṇā (OCR error)
![pata](https://cloud.githubusercontent.com/assets/80761/11287676/99ec14ac-8f2e-11e5-8e18-09bec29928b9.PNG)

**Comment by gasyoun** [comment]
pratuṣṭa -> pratuṣṭī (OCR error)
![pratuzwa](https://cloud.githubusercontent.com/assets/80761/11287697/c821bed0-8f2e-11e5-8d19-7578a9849b53.PNG)

**Comment by gasyoun** [comment]
priyacikīrṣa -> priyacikīrṣa (OCR error)
![priyacikirza](https://cloud.githubusercontent.com/assets/80761/11287730/fd0e5856-8f2e-11e5-8000-3427aea95528.PNG)

**Comment by gasyoun** [comment]
mumukṣa -> mumukṣā (OCR error)
![mumukza](https://cloud.githubusercontent.com/assets/80761/11287759/39282a74-8f2f-11e5-8962-721cc8f0fe43.PNG)

**Comment by gasyoun** [comment]
vaiśyāja (print? error), @drdhaval2785 
der Sohn einer Vaiçyâ. = son of a Vaiçyâ
![vesyaja](https://cloud.githubusercontent.com/assets/80761/11287804/78000aa0-8f2f-11e5-9530-6555d660792d.PNG)

**Comment by gasyoun** [comment]
vyavasthita [L=23746] -> vyavasthiti  (OCR error)
![vyavastita](https://cloud.githubusercontent.com/assets/80761/11287854/c8c62e56-8f2f-11e5-81d3-ef5c400b09ad.PNG)

**Comment by gasyoun** [comment]
śabdavidya -> śabdavidya (OCR error)
![sabdavidya](https://cloud.githubusercontent.com/assets/80761/11287873/f8fb66fe-8f2f-11e5-8edf-a0a6307625db.PNG)

**Comment by gasyoun** [comment]
strītva¦ f. Weiblichkeit. [=Womanhood] (print? error)
![stritva](https://cloud.githubusercontent.com/assets/80761/11287903/36f9219e-8f30-11e5-84e9-323166c08af6.PNG)

**Comment by gasyoun** [comment]
sthāvaratva¦ f. Unbeweglichkeit.
![stavaratva](https://cloud.githubusercontent.com/assets/80761/11287932/69908868-8f30-11e5-9581-4fe759bd331b.PNG)

**Comment by gasyoun** [comment]
hitopadeśa¦ f. gute Unterweisung T. eines Fabelbuchs.
![hitopadesa](https://cloud.githubusercontent.com/assets/80761/11287952/a21a2ca2-8f30-11e5-8110-b6010eacb7ea.PNG)

This ends http://www.sanskrit-lexicon.uni-koeln.de/scans/CCSScan/2014/web/webtc/indexcaller.php submissions. Few left unclear, @zaaf2 wanted.

**Comment by gasyoun** [comment]
Only `AP` case

samita f.
.²1 War, battle; samiti patinipātākarṇana &c. N. 12. 75.
.²2 Association; L. D. B.
.-Comp.
.-sajja a. ready for war.

![samita](https://cloud.githubusercontent.com/assets/80761/11287994/02202160-8f31-11e5-8e9e-673163132ae5.PNG)

**Comment by gasyoun** [comment]
2 `SCH` cases.
>kubjottarA</key1><key2>Kubjottara</key2></h><body>f. N. pr. einer Nonne , Ja1takam. 19 , 35; Divya1vad. 533 , 5; 539 , 16 ff. 

Everything is fine, false positive.

>adhoguḍā [L=1841] [p= 025-2] 	f. Argyreia speciosa , Caraka 1 , 1. 2

Everything is fine, false positive. It states `aDoguqa` in @drdhaval2785 file, but is `aDoguqA` in @funderburkjim display.

![kubj](https://cloud.githubusercontent.com/assets/80761/11288041/560edd8e-8f31-11e5-9e9c-a9499ba5db5f.PNG)

**Comment by gasyoun** [comment]
2 `GRA` cases false positives, reference articles.
`asita`
`SItaka`

**Comment by gasyoun** [comment]
5 `PWG` cases, can't decide
dāḍimba
![daqimba](https://cloud.githubusercontent.com/assets/80761/11288142/47f406e2-8f32-11e5-918f-557b792cb958.PNG)

**Comment by funderburkjim** [comment]
When you're ready for me to install corrections and no/changes from this, let me know, and maybe realize that I am finding it a bit hard to follow.

**Comment by gasyoun** [comment]
I'm ready, if @zaaf2 comments on cases where I have no comments.

**Comment by gasyoun** [comment]
gomatīsāra [L=73066] [p= 5-1390] 	 f.  Titel  -> gomatīsāra 	 Titel / f. not in scan
![gomatisara](https://cloud.githubusercontent.com/assets/80761/11313593/142e555c-8fef-11e5-95a3-2fe7566ca134.PNG)

**Comment by drdhaval2785** [comment]
>atisṛṣiṭa -> atisṛṣṭī (OCR error)

I guess you meant atisṛṣṭi

**Comment by drdhaval2785** [comment]
> pratuṣṭa -> pratuṣṭī (OCR error)

pratuṣṭi - no dIrgha. It is hrasva 'i'

**Comment by drdhaval2785** [comment]
>priyacikīrṣa -> priyacikīrṣa (OCR error)

priyacikīrṣā

**Comment by drdhaval2785** [comment]
>vaiśyāja (print? error), @drdhaval2785
der Sohn einer Vaiçyâ. = son of a Vaiçyâ

Then 'f.' should be converted to 'm'.
Not a feminine word. Son is masculine.

**Comment by drdhaval2785** [comment]
>śabdavidya -> śabdavidya (OCR error)

śabdavidyā

**Comment by drdhaval2785** [comment]
>strītva¦ f. Weiblichkeit. [Womanhood] print? error

'f.' -> 'm.'

**Comment by drdhaval2785** [comment]
>sthāvaratva¦ f. Unbeweglichkeit.

'f.' -> 'n.'

**Comment by drdhaval2785** [comment]
>hitopadeśa¦ f.

hitopadeśa¦ m.

**Comment by drdhaval2785** [comment]
>Only AP case
samita f.

samita -> samit (corroborated by 'samiti' in the explanation saptamI ekavacana of 'samit'.)

**Comment by drdhaval2785** [comment]
>2 SCH cases.
Agreed.

**Comment by drdhaval2785** [comment]
@gasyoun and @funderburkjim 
The thread is longer than what is manageable.

Let's shift both PWG cases to the next issue and treat PWG there.

**Comment by gasyoun** [comment]
PWG has only 5 cases. I would finish it here. I would split threads after 30 words. Agree?

**Comment by drdhaval2785** [comment]
Yes. Go ahead

**Comment by zaaf2** [comment]
vaiśyāja f. → m. (?)
Print error.

MW: 
* वैश्या-ज [p= 1026] : mfn. born of a वैश्य woman, the child of a वैश्य mother Mn.  ix, 151. [L=207865]
* ज 2 [p= 407] : mf(आ)n. ( √जन्) ifc. born or descended from, produced or caused by, born or produced in or at or upon, growing in, living at Mn.  MBh.  &c [L=76156]

Instead or `f. → m.`, perhaps the correction should be:

`वैश्याज¦ f. der Sohn einer Vaiçyâ.` →  `वैश्याज¦ f. आ der Sohn einer Vaiçyâ.`

In the same CCS printed page I find:

वैशन्त,¦ f. teichartig. Teich-. → वैशन्त,¦ f. इ, teichartig. Teich-.
Print error
![image](https://cloud.githubusercontent.com/assets/14730665/11318632/91e11e90-9039-11e5-8eb7-4807efa53b72.png)
 
MW: वैशन्त [p= 1026] : mf(ई)n. (fr. वेशन्त) contained in or forming a tank RV.  VS.  TS.  Br.  [L=207749]

(I wonder why this case was not included in the work-list)

**Comment by zaaf2** [comment]
strītva f. → strītva n.
Print (lexicographer) error

MW: स्त्रीत्व [p= 1260] : n. womanhood, wifehood MBh.  R.  &c [L=254975]

sthāvaratva¦ f. → sthāvaratva¦ n. 
Print (lexicographer) error

MW: स्थावर-त्व [p= 1264] : n. fixedness, immobility MBh.  [L=255603]

For the gender of words formed with the secondary suffix -त्व, v. Whitney, Sanskrit Grammar 1239:
![image](https://cloud.githubusercontent.com/assets/14730665/11318646/f639b14a-9039-11e5-9cef-36bc7cccc5d1.png)

**Comment by zaaf2** [comment]
hitopadeśa¦ f. → hitopadeśa¦ m.
Print error.

MW:
* हितो*पदेश [p= 1298] : m. friendly advice, salutary instruction MBh.  Pañcat.  &c [L=262928]
* उप-देश a [p= 199] : m. pointing out to, reference to (…); m. specification, instruction, teaching, information, advice, prescription (…)

**Comment by gasyoun** [comment]
@zaaf2 why the regex missed वैशन्त,¦ f. I hardly understand. Your ` वैश्याज¦ f. आ ` proposal makes more sense than @drdhaval2785

**Comment by gasyoun** [comment]
* yogaratnasamuccaya
f. can't be hrasva
![yogaratnasamuccaya](https://cloud.githubusercontent.com/assets/80761/11320368/a615ef80-90a7-11e5-8c5a-6155156293aa.PNG)

**Comment by gasyoun** [comment]
romodgama [L=85462] [p= 6-0449]  f. 
f. can't be hrasva 
![romodgama](https://cloud.githubusercontent.com/assets/80761/11320384/f296bf60-90a7-11e5-8491-268b74ea850f.PNG)

**Comment by gasyoun** [comment]
* vajralepa [L=87427] [p= 6-0635] 	 f.  Bez. einer -> vajralepa m.  Bez. eines (two OCR errors)

![vajralepa](https://cloud.githubusercontent.com/assets/80761/11320400/657803e0-90a8-11e5-93a9-b778c7798d46.PNG)

**Comment by gasyoun** [comment]
https://github.com/sanskrit-lexicon/CORRECTIONS/issues/166 continuing.

**Comment by funderburkjim** [comment]
Regarding the problems in ccs noticed relative to devAgAra:

21 headwords, from devasad, devasadman through devAri, devArcana were involved.

The correction was that the digitized definition for devasadman was really that of devasad,
and so forth with the digitized definition for devArcana was really that of devAri.
And, finally, the definition for devArcana was not digitized, and so was added .

I think this cleans this up.

**Comment by funderburkjim** [comment]
ccs changes installed.  There were no 'nochange' cases.

Here are the ccs changes, in standard form:
```
ccs:atisfziwa:atisfzwi:t:
ccs:arizwa:arizwi:t:L=1774
ccs:UrRa:UrRA:t:
ccs:kulapratizWa:kulapratizWA:t:
ccs:kfzwa:kfzwi:t:[L=5343]
ccs:cikyAna:cikyAna:t:f. 2. ci ->  s. 2. ci (not headword)
ccs:jijIviza:jijIvizA:t:
ccs:tvaca:tvac:t:[L=9186]
ccs:durvArtta:durvArttA:t:
ccs:nIzpada:nizpad:t:
ccs:pawahaGozaRa:pawahaGozaRA:t:
ccs:pratuzwa:pratuzwi:t:
ccs:priyacikIrza:priyacikIrzA:t:
ccs:mumukza:mumukzA:t:
ccs:vESyAja:vESyAja:t:f. -> m. (headword ok)
ccs:vyavasTita:vyavasTiti:t:[L=23746] 
ccs:Sabdavidya:SabdavidyA:t:
ccs:strItva:strItva:p:f. -> n.  (headword ok)
ccs:sTAvaratva:sTAvaratva:p:f. -> n.  (headword ok)
ccs:hitopadeSa:hitopadeSa:p:f. -> m.  (headword ok)
ccs:vESanta:vESantA:p:
```

**Comment by funderburkjim** [comment]
ap change installed.
In standard form:
```
ap:samita:samit:t:[L=33932]
```

**Comment by funderburkjim** [comment]
pwg changes installed.
In standard form:
```
pwg:dAqimba:dAqimba:t:f. -> m. (MW)  (headword ok)
pwg:putrIkaraRamImAMsAM:putrIkaraRamImAMsA:t:print smudge
pwg:hrasvagaveDukAM:hrasvagaveDukA:t:print smudge
pwg:yogaratnasamuccaya:yogaratnasamuccaya:p:f.->m. (headword ok)
pwg:romodgama:romodgama:p:f. -> m. (headword ok)
pwg:vajralepa:vajralepa:t:f.->m. and einer->eines
```

**Comment by funderburkjim** [comment]
All corrections now installed.

**Comment by gasyoun** [comment]
(headword ok) seems a bit strange comment.
Maybe look inside? How to say that what I hunt the most is right in these cases, only the grammar markup is gone wrong?
